spec/features/topic_admin_feature_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature "Topic Admin Process:", js: true do
+
+  before do
+    admin = create(:admin)
+    visit root_path
+    click_link "Login"
+    fill_in :user_email, with: admin.email
+    fill_in :user_password, with: admin.password
+    click_button "Sign in"
+    expect(current_path).to eq('/')
+    expect(page).to have_content("Signed in successfully.")
```

Comment:
The login steps are reused throughout the feature specs.

Consider moving the login code to a single helper method that can be reused in the feature specs.

```ruby
# In spec/support/login_helper.rb

module LoginHelper
  def login(user)
    click_link 'Login'
    fill_in :user_email, with: admin.email
    fill_in :user_password, with: admin.password
    click_button "Sign in"
    expect(current_path).to eq('/')
    expect(page).to have_content("Signed in successfully.")
  end
end

RSpec.configure do |config|
  # Make the LoginHelper methods available to all feature specs
  config.include LoginHelper, type: :feature
end
```

With the above login_helper.rb file, it'll be possible to call `login(user)` in feature specs to perform the login.

This reuse technique doesn't have to be limited to login. This technique of extracting a test helper method can be used for other steps that repeat in various feature specs.